src/components/PageHeader.js

```diff
@@ -1,18 +1,34 @@
+import PropTypes from 'prop-types';
+import HeaderBtn from './HeaderBtn';
 import logo from '../images/logo.svg';
 import '../styles/PageHeader.scss';
 
-export default function PageHeader({ hasBackground, headerBtn, children }) {
+export default function PageHeader({
+  hasBackground,
+  isMovieDetailsMode,
+  toggleMovieDetailsMode,
+  children,
+}) {
   return (
     <header className={`page-header ${hasBackground ? 'has-background' : ''}`}>
       <div className="page-header__panel">
         <a href="/">
           <img src={logo} alt="Netflix roulette" />
         </a>
 
-        {headerBtn}
+        <HeaderBtn
```

Comment:
HeaderBtn is a bad name for a component since Btn should not know about Header. So name of a button should only be about btn props.